Mathlib/Order/Hom/Basic.lean

```diff
@@ -522,6 +522,11 @@ protected def withTopMap (f : α →o β) : WithTop α →o WithTop β :=
 
 end OrderHom
 
+-- See note [lower instance priority]
+instance (priority := 100) OrderHomClass.toOrderHomClassOrderDual [LE α] [LE β]
+    [FunLike F α β] [OrderHomClass F α β] : OrderHomClass F αᵒᵈ βᵒᵈ where
```

Comment:
This can't be an instance because `α` and `β` are out-params in `OrderHomClass`

Reply:
It worked at my site of use here: https://github.com/leanprover-community/mathlib4/pull/15950/files#diff-6c24c5f799f540195c932d072bf980aa91eaf384ac4fecb2d64e56042aa23791R250

Reply:
Ah, I see. You're saying for `OrderHom` it won't work, while I used `OrderIso`